index.js

```diff
@@ -1170,6 +1171,17 @@ Read more on https://git.io/JJc0W`);
     return this;
   };
 
+  /**
+   * Allow excess arguments on the command line.
+   *
+   * @param {Boolean} [allowExcess] - if `true` or omitted, no error will be thrown
+   * for excess arguments.
+   */
+  allowExcessArguments(allowExcess) {
```

Comment:
Are the default parameters available?

```js
allowExcessArguments(allowExcess = true) 
```

Reply:
Good question, I'll look into that. We have a few methods with this sort of pattern that could use it. Likely can because we have moved forward our minimum version of node.

Reply:
Defaults from node 6, which is our minimum supported version: https://node.green/#ES2015-syntax-default-function-parameters

Reply:
Added default parameter instead of manual test to multiple routines.

Reply:
Thank you for fixing it.